pkg/operation/botanist/garbage_collection.go

```diff
@@ -95,3 +102,43 @@ func (b *Botanist) deleteStalePods(client kubernetes.Interface, podList *corev1.
 
 	return result
 }
+
+func (b *Botanist) removeStaleOutOfDiskNodeCondition() error {
+	// TODO: This code can be limited to 1.13.0-1.13.3 (assuming 1.13.3 has contains the Kubernetes fix) as soon as
+	// a new release containing the fix has been cut.
```

Comment:
Can you add @afritzler's PR https://github.com/kubernetes/kubernetes/pull/73394 which should be merged in `1.13.3`?

Reply:
done